src/components/ConversationSettings/ConversationPermissionsSettings.vue

```diff
@@ -38,7 +38,7 @@
 			</CheckboxRadioSwitch>
 			<span v-show="loading && radioValue === 'all'" class="icon-loading-small" />
 		</div>
-		<p>{{ t('spreed', 'Participants have permissions to start a call, join a call, enable audio, video and screenshare.') }}</p>
+		<p>{{ t('spreed', 'Participants have permissions to start a call, join a call, enable audio and video, and screenshare.') }}</p>
```

Comment:
It's a separate permission, so i would use "audio and video" and then comma separate screenshare

Reply:
Dane 😉.